Bubble sort

Ramon-Carrillo opened the pull request:
The project is to write a loop that does the same work as the .sort method in Ruby. We used two different types of loops and yield on the second loop.

tirthajyoti-ghosh reviewed (changes requested):
# Changes required 🛑 

## What a remarkable project you have built 💯! This is impressive 🤩👌🧨!

### But in order to get it approved, we need to work on some changes. Let's see...

- Your README can be improved.
  
  - Please specify the ruby version in the "Prerequisites" section.
  - Please provide descriptive instructions for the "Usage" section.
  - Since this project does not require tests, you can remove the "Run tests" section.

- In your `bubble_sort_by` method, you are not returning anything. Instead, you are `print`ing the sorted array. Please remove the `print` statement on line 31 and return the sorted array.

Your project looks absolutely fantastic. Just make these changes and apply for another code review.

Happy coding 👨‍💻 

*Need help? Shoot me a message on Slack - [@Ghosh](https://app.slack.com/client/T47CT8XPG/DRDM6LAQN/user_profile/URB1MHXK6)*
<hr>

**Tirthajyoti Ghosh**

[GitHub](https://github.com/tirthajyoti-ghosh) | [LinkedIn](https://www.linkedin.com/in/tirthajyoti-ghosh/) | [Twitter](https://twitter.com/terrific_ghosh)

Ramon-Carrillo commented:
-Added Ruby version to 2.7
-Added clear instructions on how to use the file. 
-Removed Run test from Readme
-Removed print statement from line 31 and was able to return the sorted array.

maelfosso reviewed (approved):
# Congratulations :clap: 

Thank you for making the suggested changes
You can merge it now :+1: 

Good continuation :handshake: 

Mael FOSSO
